README.md

```diff
@@ -401,17 +224,22 @@ The Julia source code is organized as follows:
     contrib/       editor support for Julia source, miscellaneous scripts
     deps/          external dependencies
     doc/src/manual source for the user manual
-    doc/src/stdlib source for standard library function reference
     src/           source for Julia language core
     test/          test suites
     ui/            source for various front ends
     usr/           binaries and shared libraries loaded by Julia's standard libraries
 
 ## Binary Installation
 
-If you would rather not compile the latest Julia from source, platform-specific tarballs with pre-compiled binaries are also [available for download](https://julialang.org/downloads/).
+If  you  would  rather  not  compile the  latest  Julia  from  source,
+platform-specific  tarballs   with  pre-compiled  binaries   are  also
+[available for download](https://julialang.org/downloads/).
+
+You can either run the `julia` executable using its full path in the
```

Comment:
Probably also unnecessary text. The download page provides instructions.

Reply:
Good point. Can remove.